generator/src/test/java/com/scottlogic/deg/generator/generation/FieldSpecValueGeneratorTests.java

```diff
@@ -83,12 +86,12 @@ void generate_fieldSpecContainsSingleMustContainRestriction_returnsValuesWithVal
             )
         );
 
-        assertEquals(expectedDataBags, result);
+        assertThat(result, sameBeanAs(expectedDataBags));
```

Comment:
asserts should be `assert(expected, actual)` otherwise when it fails it says it expected what it actual got
https://junit.org/junit4/javadoc/4.12/org/junit/Assert.html

Reply:
Sorry @mrspaceman, assertThat is the inverse order:
https://junit.org/junit4/javadoc/4.12/org/junit/Assert.html#assertThat(java.lang.String,%20T,%20org.hamcrest.Matcher)

```
assertThat(T actual, Matcher<? super T> matcher)
```